eo/Dockerfile

```diff
@@ -17,3 +17,7 @@ ENV CPL_ZIP_ENCODING=UTF-8
 
 # specify user because of problem running start-notebook.sh when being root
 USER jenkins
+
+# Add a copy of the eo notebooks to the docker image
+# All tutorial-notebooks are set with jenkins as owner, so they can be manipulated by jenkins in the docker entrypoint
+COPY --chown=jenkins notebooks/ /notebook_dir/tutorial-notebooks
```

Comment:
I am surprised notebooks are copied directly into the image.  The the other associated PR https://github.com/bird-house/pavics-jupyter-base/pull/2 you just wrote "avoid having to rebuild the image just to update a notebook".

I thought I would see this child image leverage the new feature in the base image.

Reply:
Adding to what I said here : https://github.com/bird-house/pavics-jupyter-base/pull/2#issuecomment-814238353, we looked for an easy way to add notebooks to the ones provide in the base image. We agree that having a custom image with static notebooks is the bare minimum, but it fits our needs right now. We do not expect a lot of modifications in a near future and until we build new images weekly, having another repository just for some notebooks, seems overkill. Adding a wget in this image is in the plan given that the notebooks start to be updated more frequently.